Pipeline/SimplePackageDeploy/SimplePackageDeploy.groovy

```diff
@@ -0,0 +1,276 @@
+
+def rc = runProcess(processCmd)
+if (rc == 0) println("Package untar done to $tarExtractDirName")
+else {
+	println("Failed to untar $tarFile")
+	System.exit(1)
+}
+
+
+// Check if BuildReport.json exists, deploy files if exists and if not stop deploy 
+
+def buildReportFile = new File("${tarExtractDirName}/BuildReport.json")
+if (buildReportFile.exists()) {	
+	deployFromBuildReport(buildReportFile,tarExtractDirName,targetLibLLQMap,copyModeMap)
+} else {
+	println("** Build report data at $tarExtractDirName/BuildReport.json not found")
+	println("** Deployment stopped")
+}
+
+if (tarExtractDir.exists()) tarExtractDir.deleteDir()
+println("\nCleaning up the temporary folder - ${tarExtractDirName}\n")
+
+/**********************************************************************************
+ **** Deploy from the BuildReport                  
+ **********************************************************************************/
+def deployFromBuildReport (File buildReportFile, String tarExtractDirName,Map targetLibLLQMap,Map copyModeMap) {
+	
+	println("\n** Deploying the contents in ${buildReportFile}")
+	
+	def buildReport= BuildReport.parse(new FileInputStream(buildReportFile))
+	def tarExtractDir = new File("${tarExtractDirName}")
+
+	//finds all the build outputs with a deployType
+	def executes= buildReport.getRecords().findAll{
+	 try {
+	     (it.getType()==DefaultRecordFactory.TYPE_EXECUTE &&
+	             !it.getOutputs().isEmpty())
+	 } catch (Exception e){}
+	}
+
+
+	//New CopyToPDS instance 
+	CopyToPDS copy = new CopyToPDS()
+
+	//Copy the extracted files to the corresponding target PDS
+	executes.each { 
+	 it.getOutputs().each {
+	     if (it.deployType != null) {
+	         def (dataset, member) = it.dataset.split("\\(|\\)")
+	         def srcFilePath = tarExtractDirName + "/" + dataset + "/" + member		
+	         if (props.packageWithExtension && props.packageWithExtension.toBoolean()) {	
+	         	srcFilePath = srcFilePath + ".${it.deployType}"
+	         }
+	         
+	         def targetPDS = "${props.hlq}" + "." + targetLibLLQMap["${it.deployType}"]
+	         println("\nExtracted file $srcFilePath is of type ${it.deployType}") 
+	         if (copyModeMap["${it.deployType}"] != null) {
+	             copy.setCopyMode(DBBConstants.CopyMode.valueOf(copyModeMap["${it.deployType}"]))
+	         
+	             def srcFile = new File("$srcFilePath")
+	             if(!srcFile.exists()){
+	                 println("** Source file not found at $srcFilePath")
+	                 println("** Validate if the package was generated with addExtension option. If yes, use option: -e or --packageWithExtension to deploy")
+	                 System.exit(1)
+	             }
+	             copy.setFile(srcFile);
+
+	             copy.setDataset("$targetPDS");
+	             copy.setMember("$member");
+	             copy.copy()
+	         
+	             println("Copied source file - $dataset/$member to Target PDS - $targetPDS")
```

Comment:
Let's indent the outputs a bit for better reading of the outputs:

Also, I would like to suggest to rephrase the output as:
`Copied file xxx to target library yyy using DBB CopyMode zzz`

```
** Deploying the contents in /u/dbehm/git/anup/work/DeployFiles_20221111.025913.059/BuildReport.json
     Extracted file /u/dbehm/git/anup/work/DeployFiles_20221111.025913.059/DBEHM.DBB.BUILD.DBRM/EPSCMORT is of type DBRM
     Copied source file - DBEHM.DBB.BUILD.DBRM/EPSCMORT to Target PDS - DBEHM.DBB.BUILD.MORTGAGE.DBRM
```

Reply:
Done